ruby/vampires.rb

```diff
@@ -56,5 +56,4 @@
   end

```

Comment:
Thank you for the review, Tyler! I had my doubts about where in the code I should place the allergy part of the challenge. I guess I should've just asked for a clarification. I will take your advice and refactor my code.